policies/queries/dynamodb/table_global_secondary_indexes_unused.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'                                                   as execution_time,
+       :'framework'                                                        as framework,
+       :'check_id'                                                         as check_id,
+       'DynamoDB global secondary indexes with no items should be removed' as title,
+       account_id,
+       arn                                                                 as resource_id,
+       case
+           when item_count > 0 then 'pass'
+           else 'fail'
+           end                                                             as status
```

Comment:
I don't think we can make this recommendation. It is a valid use case to have empty DynamoDB indexes

Reply:
IMO, you need the index to lookup some set of values fast.
If that's something extraordinary (e.g., errors that are processed once they appear with happy path producing no errors at all), I can remove this query.

Reply:
Yes, for tables that have both write and delete operations it is possible for the index or table to be empty... For example I have used DynamoDB as a priority queue before... Where a consumer hits an index to see if there are any high priority items otherwise it hits the main table...

Reply:
OK, removed the indexes then